### ISSUE TITLE
Create one auto-generated Probeneinsendebogen per NRL on send

### ISSUE THREAD

**Issue by afalenski** [SiLeBAT/mibi-portal-server#131]
As a user I would like the portal to create one Probeneinsendebogen per NRL. 
- [ ] Upon send split data into NRL-specific files
- [ ] Use the contact details from the uploaded Probeneinsendebogen.
- [ ] Change the lab according to the pathogen in the cell next to "Labor".
- [ ] Attach Excel sheet to the email sent to BfR and to the email sent to the user.

**Comment by GerberI** [comment]
- [ ] Upon send split data into NRL-specific files
- [ ] Change "Labor" in each NRL-specific file

**Comment by afalenski** [comment]
Testsheet with agents for BfR labs used. 
- 11 out of 11 BfR labs were labelled correctly (Leptospira and Trichinella could not pass because of non-ADV agent names. Thus, both were not included. Will be re-tested.
- 11 Excel sheets were generated and sent to the user.
- 11 emails with one excel sheet each were sent to the 11 BfR lab addresses.

**Comment by GerberI** [comment]
Perhaps we should handle the first two issues: missing "Ort, Datum" and missing "human isolates/sonstiges" in a separate ticket after #258 is done. The reason is that these points touches the client, api and server and i think is big enough to separate it. Lets discuss this tomorrow.

**Comment by GerberI** [comment]
@Alex
Can u send me a test sheet with pathogens for Bacillus spp. and Clostridium spp. so that i can test the 3rd issue?

**Comment by afalenski** [comment]
Issue 3 is now correct.

**Comment by dptolle** [comment]
@afalenski @GerberI Can this Ticket be placed into Done?
